feat: deprecate addons command (#6123)
netlify/cli: docs/commands/addons.md

## About

> > ⚠️ Addons has been deprecated and will become unavailable in a future update.

Reviewer: Wondering if we can use the same language as our release phases (`end of service` instead of `unavailable`) and link to the page: https://docs.netlify.com/platform/release-phases/#deprecated

I'm including a slight grammar edit here also:

```suggestion
> ⚠️ The addons command is [deprecated](https://docs.netlify.com/platform/release-phases/#deprecated) and will be end of service in a future update.
```

Though I see we used a similar message for large media, so I'm ok with just adding a link to the page but leaving the rest the same

```suggestion
> ⚠️ The addons command is [deprecated](https://docs.netlify.com/platform/release-phases/#deprecated) and will become unavailable in a future update.
```
